lib/absinthe/phase/schema.ex

```diff
@@ -237,6 +240,14 @@ defmodule Absinthe.Phase.Schema do
           Absinthe.Schema.t(),
           Absinthe.Adapter.t()
         ) :: nil | Type.Field.t()
+  defp find_schema_field(%{fields: fields}, "__inputname" = name, node, schema, adapter) do
+    internal_name = adapter.to_internal_name(name, :field)
+
+    fields
+    |> Map.values()
+    |> Enum.find(&match?(%{name: ^internal_name}, &1))
```

Comment:
necessary

and I prefer to keep the logic duplicated

Reply:
might be `:string` every time